R/input-text.R

```diff
@@ -10,6 +10,14 @@
 #' @param placeholder A character string giving the user a hint as to what can
 #'   be entered into the control. Internet Explorer 8 and 9 do not support this
 #'   option.
+#' @param ... Ignored, included to require named arguments and for future
+#'   feature expansion.
+#' @param updateOn A character vector specifying when the input should be
+#'   updated. Options are `"change"` (default) and `"blur"`. Use `"change"` to
+#'   update the input immediately whenever the value changes. Use `"blur"`to
+#'   delay the input update until the input loses focus (the user moves away
+#'   from the input), or when Enter is pressed (or Cmd/Ctrl + Enter for
+#'   [textAreaInput()]).
```

Comment:
I wonder if some people will want to have a third option, `"enter"`? In that scenario, we'd allow multiple values for `updateOn`, so you could say `updateOn = c("blur", "enter")`.

That said, I prefer the implementation as it is now; I don't think in practice in Shiny apps it's a great idea to require pressing Enter in these inputs for the value to change, or if you do you should use an button (or we extract the shinychat text input into a standalone component).